scripts/get_scarb_versions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+
+# This script is used to find the following Scarb versions:
+# The current major.minor version along with all its patch versions
+# The latest patch versions for the two versions preceding the current one
+
+function get_all_patch_versions() {
+  # Omit version 2.8.0 as it has a bug when using the `assert_macros` package
+  asdf list all scarb $1 | grep -v "rc" | grep -v "2.8.0"
```

Comment:
```suggestion
  asdf list all scarb "$1" | grep -v "rc" | grep -v "2.8.0"
```

Please always quote your variables to prevent globbing and/or splitting 🙏 (applicable to the rest fo the code as well)